openedx/core/djangoapps/course_groups/tests/test_views.py

```diff
@@ -896,9 +902,32 @@ def test_only_unknown(self):
             expected_added=[],
             expected_changed=[],
             expected_present=[],
+            expected_preregistered=[],
             expected_unknown=usernames
         )
 
+    def test_preregistered_users(self):
+        """
+        Verify that email addresses can be preregistered for a cohort if the user associated with that email
+        address does not yet exist.
+        """
+        email_addresses = ["email@example.com", "email2@example.com", "email3@example.com"]
+        response_dict = self.request_add_users_to_cohort(
```

Comment:
I'm confused how this test is working because it seemed like you were only keeping track of count, not email addresses.